carsons.egg-info/PKG-INFO

```diff
@@ -0,0 +1,323 @@
+Metadata-Version: 2.1
```

Comment:
can we remove everything in `carsons.egg-info` from the MR? i think this stuff should be autogenerated during the packaging step and we don't want it. you might even add `carsons.egg-info/*` to the .gitignore

Reply:
Yes...that slipped by me